pkg/registry/core/podtemplate/strategy.go

```diff
@@ -77,7 +86,12 @@ func (podTemplateStrategy) PrepareForUpdate(ctx context.Context, obj, old runtim
 func (podTemplateStrategy) ValidateUpdate(ctx context.Context, obj, old runtime.Object) field.ErrorList {
 	template := obj.(*api.PodTemplate)
 	oldTemplate := old.(*api.PodTemplate)
-	return corevalidation.ValidatePodTemplateUpdate(template, oldTemplate)
+
+	// Allow downward api usage of hugepages on pod update if feature is enabled or if the old pod already had used them.
+	opts := validation.PodValidationOptions{}
```

Comment:
use unified helper to get opts (comment applies to all strategy computation of opts)